menas/src/main/resources/application.properties.template

```diff
@@ -120,3 +123,15 @@ za.co.absa.enceladus.menas.oozie.splineMongoURL=mongodb://localhost:27017
 #In some cases extra driver options need to be surrounded by double quotes
 #This seems to depend of the version of oozie
 #za.co.absa.enceladus.menas.oozie.sparkConf.surroundingQuoteChar="
+
+#Configure the following three parameters to use oozie impersonation
+#za.co.absa.enceladus.menas.oozie.proxyUser=oozieProxyUser
+#za.co.absa.enceladus.menas.oozie.proxyGroup=oozieProxyGroup
+#za.co.absa.enceladus.menas.oozie.proxyUserKeytab=/path/to/keytab
+
+#The following property allows customization of the error message presented to the user if they're missing the impersonation group
+#za.co.absa.enceladus.menas.oozie.customImpersonationExceptionMessage="Please add the system user into the '${za.co.absa.enceladus.menas.oozie.proxyGroup}' group to use this feature."
```

Comment:
Does the embedded SpEL (`${za.co.absa.enceladus.menas.oozie.proxyGroup}`) here work? Won't it just print as `Please add the system user into the '${za.co.absa.enceladus.menas.oozie.proxyGroup}' group to use this feature.`?

Reply:
Yeah this double checked this now, works fine.

Reply:
Interesting!